inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/ExtractNode.java

```diff
@@ -62,7 +63,8 @@
         @JsonSubTypes.Type(value = TubeMQExtractNode.class, name = "tubeMQExtract"),
         @JsonSubTypes.Type(value = PulsarExtractNode.class, name = "pulsarExtract"),
         @JsonSubTypes.Type(value = RedisExtractNode.class, name = "redisExtract"),
-        @JsonSubTypes.Type(value = DorisExtractNode.class, name = "dorisExtract")
+        @JsonSubTypes.Type(value = DorisExtractNode.class, name = "dorisExtract"),
+        @JsonSubTypes.Type(value = InfluxDBExtractNode.class, name = "influxDBExtract")
```

Comment:
Please keep the naming consistent of influxExtract between the Node and ExtractNode.